### ISSUE TITLE
Vulnerable dependencies

### ISSUE THREAD

**Issue by dicearr** [wearereasonablepeople/webpacker#55]
There are still 7 vulnerabilities: 3 low and 4 moderate.

- `favicons-webpack-plugin` has 3 vulnerable dependencies. Waiting for [this issue](https://github.com/jantimon/favicons-webpack-plugin/issues/132).
- `node-sass`  has 4 vulnerable dependencies. Waiting for [this issue](https://github.com/sass/node-sass/issues/2355).

**Comment by renanborgez** [comment]
https://github.com/sass/node-sass/issues/2355#issuecomment-411749402 `node-sass` Resolved in 4.9.3

**Comment by Avaq** [comment]
I'm using webpacker and get the following report from `npm audit`:

```text
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ webpacker                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ webpacker > favicons-webpack-plugin > favicons > cheerio >   │
│               │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ webpacker                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ webpacker > favicons-webpack-plugin > favicons >             │
│               │ merge-defaults > lodash                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ debug                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >= 2.6.9 < 3.0.0 || >= 3.1.0                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ webpacker                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ webpacker > favicons-webpack-plugin > favicons >             │
│               │ node-rest-client > debug                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/534                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

According to comments on https://github.com/jantimon/favicons-webpack-plugin/issues/132, the library has been forked to an actively maintained version. We should consider switching to that.
